tests/test_helper/bats/services.bats

```diff
@@ -0,0 +1,50 @@
+
+load "${ABS_TOP_TEST_DIRNAME}test_helper/setup_teardown/$(basename "${BATS_TEST_FILENAME//.bats/.bash}")"
+
+setup() {
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/common.bash
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/lxd.bash
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/microovn.bash
+    load ${ABS_TOP_TEST_DIRNAME}../.bats/bats-support/load.bash
+    load ${ABS_TOP_TEST_DIRNAME}../.bats/bats-assert/load.bash
+
+    # Ensure TEST_CONTAINERS is populated, otherwise the tests below will
+    # provide false positive results.
+    assert [ -n "$TEST_CONTAINERS" ]
+}
+
+services_register_test_functions() {
+    bats_test_function \
+        --description "Testing of service functionality" \
+        -- service_tests
+}
+
+service_tests() {
+    for container in $TEST_CONTAINERS; do
+        # enable enabled service
+        run lxc_exec "$container" "microovn enable switch"
+        assert_output "Error: Failed to enable service 'switch': 'Service already exists'"
+
+        # enable non existing service
+        run lxc_exec "$container" "microovn enable switchh"
+        assert_output "Error: Failed to enable service 'switchh': 'Service does not exist'"
+
+        # disable non existing service
+        run lxc_exec "$container" "microovn disable switchh"
+        assert_output "Error: Failed to disable service 'switchh': 'This service is not enabled'"
+
+        # disable service
+        run lxc_exec "$container" "microovn disable switch"
+        assert_output "Service switch disabled"
```

Comment:
I forgot to mention this in my last review, but I think that the test should verify more than just command output. E.g:
* Check `microovn status` output to make sure that it reports expected services on the node
* Check `snap services microovn` to make sure that expected services were actually started